Cargo.toml

```diff
@@ -15,10 +15,12 @@ categories = ["email", "network-programming"]
 
 [features]
 tls = ["native-tls"]
+rustls-tls = ["rustls-connector"]
```

Comment:
Separately, I wonder if we want to take this opportunity to rename the "other" `tls` feature `native-tls`, since it isn't really "more" standard than `rustls`.

Reply:
I think this is a good idea, so did a PR up [here](https://github.com/jonhoo/rust-imap/pull/201).